actionpack/lib/action_dispatch/middleware/debug_exceptions.rb

```diff
@@ -19,7 +21,7 @@ def self.register_interceptor(object = nil, &block)
     end
 
     def initialize(app, routes_app = nil, response_format = :default, interceptors = self.class.interceptors)
-      @app             = app
+      @app             = ActionableExceptions.new(app)
```

Comment:
I see why this is here, but it still troubles me that it's not being installed by the main middleware stack.

Reply:
Tried to hide it a bit, yeah... I didn't want to have the implementation in `DebugExceptions`, but also didn't wanted to have yet another `*Exceptions` middleware in the default stack. Moved it to the default stack so it's at least explicit.